Documentation/GeneralConventions/ReviewPolicy.rst

```diff
@@ -0,0 +1,201 @@
+contributors and maintainers of the documentation.
+
+With this document, we try to set up conventions and best practices.
+We welcome constructive feedback to this living document with suggestions
+to continuously improve our process.
+
+.. _review-policy-contributors:
+
+Policy for Contributors
+=======================
+
+As outlined in the chapter :ref:`docs-contribute`, contributors can
+provide so called "Pull Requests" (PRs) to our manual repositories
+on GitHub.
+
+When a Pull Request is made, we kindly ask contributors to note
+the following:
+
+*   Please ensure that your Pull Request is **made against the appropriate
+    version of the documentation**, usually against the `main` branch. If the
+    changes you propose apply to specific versions of the documentation,
+    you can add a line like `Releases: main, 12.4, 11.5` to note any
+    Git branch that your change would apply to. If you are unsure,
+    our maintainers will take care of it - this is just a step to make
+    their work easier.
+*   Please provide a **meaningful title** to your Pull Request.
+*   Please provide a **short description** of what your Pull Request
+    is about.
+*   Please try to **focus your Pull Request on a single issue/topic**. At best,
+    a pull request should only affect a single file or chapter, maybe even
+    a section. Feel free to contribute multiple Pull Requests for multiple
+    topics. This makes our review process quicker and easier.
+*   **Be open to feedback** our maintainers may have about the change,
+    and sometimes there will be questions or other comments that will help
+    to finalize your contribution.
+*   Please **be patient for feedback** to your contribution. Our maintainers
+    work on Pull Requests in their spare time.
+
+If you are interested in the work of our maintainers, read on in the
+next section.
+
+.. _review-policy-maintainers:
+
+Review policy for maintainers
+-----------------------------
+
+This section is intended for current maintainers, but please read on,
+if you are interested in maybe becoming one in the future.
+
+**Contributions are very valuable**: Anyone putting in the effort and time
+to help us should **feel rewarded** by doing so. We try to adjust our review
+policy to making this possible without sacrificing the **quality of our
+documentation**.
+
+The review process should have the **same standard** for anyone of the
+maintainer team performing the review, and every member of the team
+should feel good when **making decisions** on what and how to give feedback on.
+
+Basic work on contributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*   **Be kind and understanding**. Even when rejecting a contribution,
+    there is a person behind a text, who meant well.
+*   Try to **improve and explain changes** made to Pull Requests.
+*   The automatic **Code-Checks** must be green (maintainers may need to
+    execute them first for first-time contributors - before you do, ensure
```

Comment:
WOW...that is a good point. Never though on that. Huiiii

Reply:
Yeah, you can do nasty stuff there